app/views/hub.nunjucks

```diff
@@ -17,7 +17,7 @@
       {% if gp.contact.telephone %}Telephone: {{ gp.contact.telephone }}<br />{% endif %}
       {% if gp.contact.email %}Email: {{ gp.contact.email }}<br />{% endif %}
       {% if gp.contact.fax %}Fax: {{ gp.contact.fax }}<br />{% endif %}
-      <a href="https://www.google.com/maps?q={{ gp.location.latitude }},{{ gp.location.longitude }}">See this surgery on a map</a>
+      <a href="https://www.google.com/maps/dir/{{ gp.name | safe | replace(" ", "+")}},{{ gp.address.addressLines | safe | replace(" ", "+") }},{{ gp.address.postcode | safe | replace(" ", "+")}}/@{{ gp.location.latitude }},{{ gp.location.longitude }},17z">See this surgery on a map</a>
```

Comment:
> Can this link be generated outside of the view and added to the view model

Can you point me in the right direction please. Maybe some doco or a handy hint? I tried yesterday and didn't quite achieve it.

> I'd suggest something a bit simpler e.g. https://maps.google.com/maps?q=<name, address, postcode

Agreed and implemented.